daemon/prune.go

```diff
@@ -154,6 +156,10 @@ func (daemon *Daemon) VolumesPrune(ctx context.Context, pruneFilters filters.Arg
 
 	err = daemon.traverseLocalVolumes(pruneVols)
 
+	if err == errPruneCancelled {
```

Comment:
Thanks for your feedback. @cpuguy83 
Currently I have updated the PR and only checked `err == context.Cancelled`. And if we also check context.DeadlineExceeded, what should we return, a report with nil, or a report with error?